cabal.project

```diff
@@ -31,10 +31,21 @@ source-repository-package
 
 source-repository-package
     type: git
-    location: https://github.com/ghcjs/jsaddle
-    tag: 9cb3534b6b2dc948c6fa88fce281e59e904b43e4
-    --sha256: 1drgrgjnxamq8vh6mrqg62xcj7vb9makpy89bls485d6ckigv3ha
-    subdir: jsaddle
+    location: https://github.com/input-output-hk/hjsonpointer
+    tag: bb99294424e0c5b3c2942c743b545e4b01c12ce8
+    --sha256: 11z5s4xmm6cxy6sdcf9ly4lr0qh3c811hpm0bmlp4c3yq8v3m9rk
+
+source-repository-package
```

Comment:
Please add comments explaining what these are for and when they can be removed.

Reply:
Explanations added.